Dockerfile

```
@@ -2,8 +2,10 @@ FROM python:3.11-alpine
 
 # Install package
 WORKDIR /code
-COPY . .
-RUN pip3 install .
+COPY pyproject.toml pdm.lock README.md ./
+COPY qbittorrent_exporter ./qbittorrent_exporter
+RUN pip install . \
+    && rm -rf /root/.cache/pip
```

I think this has the same effect WDYT?

```suggestion
RUN pip install --no-cache-dir .
```

---

It should be but I inspected the resulting layer in the Docker image and `--no-cache-dir` creates one file.

---

Just one file? How big is it? I think that's not much and definitely nothing important...